src/api/repos/user-repo.ts

```diff
@@ -1,54 +1,50 @@
-import { User } from '../models/User';
-import { httpStatusCode } from '../../types/httpStatusCodes';
-import { logger } from '../../config/logger';
-import { APIError } from '../../errors/api-error';
-import { BaseError } from '../../errors/base-error';
+import {User} from '../models/User';
+import {httpStatusCode} from '../../types/httpStatusCodes';
+import {logger} from '../../config/logger';
+import {APIError} from '../../errors/api-error';
+import {BaseError} from '../../errors/base-error';
```

Comment:
nit: I think you have a bad formatting here. If you look at all other files there is a space after the { and before the }.
```
import { User } from '../models/User';
import { httpStatusCode } from '../../types/httpStatusCodes';
import { logger } from '../../config/logger';
import { APIError } from '../../errors/api-error';
import { BaseError } from '../../errors/base-error';
```